WeakAuras/BuffTrigger2.lua

```diff
@@ -71,6 +71,8 @@ local timer = WeakAuras.timer
 local BuffTrigger = {}
 local triggerInfos = {}
 
+local TTC_events = Private.TTC_events -- trigger to custom
```

Comment:
Replace local TTC* with camelCase `triggerToCustomEvents` `triggerToCustomQueue` and PascalCase for subvariables like `Private.TTC_events` => `Private.TriggerToCustomEvents`
Weakauras code is not very consistent `trigger_to_custom_events` works too, at least avoid non-descriptive variable that require to find comment explaining what it is

Reply:
Yeah, seemed overly long, but you're right about TTC being non-descriptive of course